developer/appdev/articles/01-ui-development/02-angular-widget/02-migrating-an-angular-app-to-product.markdown

```diff
@@ -202,14 +205,14 @@ Follow these steps:
 10. If you're building on Windows, set `"process-serially": true` in your 
     bundle's `.npmbundlerrc` file. 
  
-11. Finally, deploy your portlet bundle:
+11. Finally, deploy your bundle:
 
         npm run deploy
 
 Congratulations! Your Angular app is deployed and now available as a widget that
 you can add to site pages. 
 
-The liferay-npm-bundler confirms the deployment:
+The Liferay NPM Bundler confirms the deployment:
```

Comment:
@jhinkey we've always referred to it as `liferay-npm-bundler` in the docs. That is the npm package name. If you think Liferay npm Bundler sounds better, that's fine, but we should update it across all the docs. If we do go this route, never capitalize npm, as they don't capitalize it themselves.

Reply:
@mwilliams2014 It's awkward to refer to Liferay Bundler Generator in regular text and then the bundler by package name liferay-npm-bundler. I agree to use lowercase npm. I'll update that.

Reply:
Sounds good. I'll make the change across the rest of our docs